src/either/either.spec.ts

```diff
@@ -53,4 +53,38 @@ describe('Either', () => {
     it('reports itself as a right', () => expect(right.isRight()).toBe(true));
     it('can be converted to a Some', () => expect(right.toMaybe().isNone()).toBe(false));
   });
+
+  describe('fromNillable', () => {
```

Comment:
well `fromNullable` doesn't make sense here because I'm including `undefined`. Could have called it `fromNullOrUndefinedable`. Stole the term `nil` from Ramda's `R.nil`.